2018-01-16T22:59:25.314Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '_iti12voyb' in collection with ID 'Asset:org.acme.biznet.Commodity' does not exist)
    at /Users/hannahredler/repos/CodeWorks/solo-project/time-is-money/server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-01-16T22:59:25.316Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '_iti12voyb' in collection with ID 'Asset:org.acme.biznet.Commodity' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '_iti12voyb' in collection with ID 'Asset:org.acme.biznet.Commodity' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/Users/hannahredler/repos/CodeWorks/solo-project/time-is-money/server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
